apps/emqx_plugins/src/emqx_plugins_app.erl

```diff
@@ -27,8 +27,8 @@
 
 start(_Type, _Args) ->
     %% load all pre-configured
-    ok = emqx_plugins:ensure_started(),
     {ok, Sup} = emqx_plugins_sup:start_link(),
+    ok = emqx_plugins:ensure_started(),
```

Comment:
Maybe try catch and log this on start.
we can't let plugin start failed lead emqx start failed.
```
Erlang/OTP 26 [erts-14.2.1] [emqx] [64-bit] [smp:8:8] [ds:8:8:8] [async-threads:4] [jit:ns]

Listener tcp:default on 0.0.0.0:1883 started.
Listener ssl:default on 0.0.0.0:8883 started.
Listener ws:default on 0.0.0.0:8083 started.
Listener wss:default on 0.0.0.0:8084 started.
Listener tcp:default on 0.0.0.0:1883 stopped.
Listener ssl:default on 0.0.0.0:8883 stopped.
Listener ws:default on 0.0.0.0:8083 stopped.
Listener wss:default on 0.0.0.0:8084 stopped.
Kernel pid terminated (application_controller) ("{application_start_failure,emqx_machine,{{shutdown,{failed_to_start_child,emqx_machine_boot,{'EXIT',{{failed_to_start_app,emqx_plugins,{emqx_plugins,{bad_return,{{emqx_plugins_app,start,[normal,[]]},{'EXIT',{#{function => fun emqx_plugins:do_ensure_started/1,errors => [{\"emqx_xxx-1.2.2\",#{reason => {bad_return,{{emqx_xxx_app,start,[normal,[]]},{'EXIT',{{badkey,<<\"auth_url\">>},[{erlang,map_get,[<<\"auth_url\">>,#{}],[{error_info,#{module => erl_erts_errors}}]},{emqx_xxx_app,config,0,[{file,\"/Users/zw/emqx/emqx_xxx/src/emqx_xxx_app.erl\"},{line,54}]},{emqx_xxx_app,start,2,[{file,\"/Users/zw/emqx/emqx_xxx/src/emqx_xxx_app.erl\"},{line,25}]},{application_master,start_it_old,4,[{file,\"application_master.erl\"},{line,293}]}]}}}},error_msg => \"failed_to_start_plugin_app\",app => emqx_xxx,err_app => emqx_xxx}}]},[{emqx_plugins,for_plugins,1,[{file,\"emqx_plugins.erl\"},{line,1016}]},{emqx_plugins_app,start,2,[{f
```